app/assets/javascripts/ratyrate.js.erb

```diff
@@ -0,0 +1,62 @@
+$.fn.raty.defaults.half = false;
```

Comment:
The gem seems to have included it by itself. I am wondering the same. Although, I'll give it a check if the gem still works the same when I get rid of it.